pkg/operator/controllers/monitoring/monitoring_controller.go

```diff
@@ -84,8 +84,7 @@ func (r *Reconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
 	// TODO(mj): controller-runtime master fixes the need for this (https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/reconcile/reconcile.go#L93) but it's not yet released.
 	ctx := context.Background()
 
-	// check feature gates and if set to false remove any persistence
-	cluster, err := r.arocli.AroV1alpha1().Clusters().Get(ctx, arov1alpha1.SingletonClusterName, metav1.GetOptions{})
+	_, err := r.arocli.AroV1alpha1().Clusters().Get(ctx, arov1alpha1.SingletonClusterName, metav1.GetOptions{})
```

Comment:
I might be missing something. What does this get gives us now?

Reply:
Nothing really.  Before we were using it to see if the feature flag was on the cluster object.  I can delete this line. @mjudeikis